### DISCUSSION TITLE
关于freebsd平台的aliyundrive-webdav

### DISCUSSION THREAD

**Discussion by skyseedtb** [messense/aliyundrive-webdav#240] [General]
请问有计划开发一个freebsd平台的aliyundrive-webdav的软件吗？

**Comment by messense** [comment]
可以先自行编译：cargo build —release

**Reply by messense** [reply]
试试 [aliyundrive-webdav-freebsd.tar.gz](https://github.com/messense/aliyundrive-webdav/files/7827399/aliyundrive-webdav-freebsd.tar.gz)
